src/main/java/org/embulk/output/GcsOutputPlugin.java

```diff
@@ -78,6 +88,26 @@ public TransactionalFileOutput open(TaskSource taskSource, final int taskIndex)
         return new GcsTransactionalFileOutput(task, client, taskIndex);
     }
 
+    private void deleteFiles(PluginTask task)
+    {
+        logger.info("delete files");
+        Storage client = createClient(task);
+        try {
+            Objects listResult = client.objects().list(task.getBucket()).setDelimiter("/").setPrefix(task.getPathPrefix()).execute();
```

Comment:
Could you use retry with exponential backoff when plugin sends request to Google API?
https://github.com/embulk/embulk-output-gcs/blob/v0.4.4/src/main/java/org/embulk/output/GcsTransactionalFileOutput.java#L167

Reply:
ok, thanks.